src/uniswap.ts

```diff
@@ -442,7 +505,52 @@ const incentivesDistributionReport = async (
     incentivesTotal
   );
 
-  printIncentivesDistribution(distributions, dustLevel);
+  let outFd: FileHandle | undefined;
+  const out = await (async (): Promise<Console> => {
+    if (outputPath == "-" || outputPath == "") {
+      return console;
+    } else {
+      outFd = await open(outputPath, "w");
+      return new console.Console(outFd.createWriteStream(), process.stderr);
+    }
+  })();
+
+  const providers: [string, number][] = Object.entries(
```

Comment:
`providers` is not getting used below - probably it's safe to delete it?

Reply:
Leftover from a debugging session.  Removed.